src/main/java/appeng/core/Registration.java

```diff
@@ -338,9 +327,9 @@ void postInit( final FMLPostInitializationEvent event )
 	{
 		this.registerSpatial( true );
 
-		final IAppEngApi api = AEApi.instance();
+		final Api api = Api.INSTANCE;
```

Comment:
We should always use the public API. It is an additional test to verify it works.